### PR TITLE
Bug 533213 - rdf:Seq deserialization in JMH contains no elements

### PR DESCRIPTION
**Pull request by neormx**

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=533213

**Comment by berezovskyi** [comment]
@jamsden @jadelkhoury @yashkhatri can anyone of you give me a +1 too? Just wanted a second opinion.

File: org.eclipse.lyo.oslc4j.build/pom.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <jena.version>3.6.0</jena.version>
```
Comment (jadelkhoury) [inline org.eclipse.lyo.oslc4j.build/pom.xml]: I assume we will wait with this merge into master until we migrate to 3.6.0 then.

Comment (berezovskyi) [reply org.eclipse.lyo.oslc4j.build/pom.xml]: No, this patch *is* the migration to 3.6.0.

Am I clear to merge?

Comment (berezovskyi) [reply org.eclipse.lyo.oslc4j.build/pom.xml]: sure

**Comment by berezovskyi** [comment]
@neormx all the blocking patches have been merged. we are almost ready to merge yours – but now it has a merge conflict. could you please try to resolve it?

**Comment by neormx** [comment]
Done.

**Comment by berezovskyi** [comment]
Ricardo, please review my last commit https://github.com/eclipse/lyo.core/pull/14/commits/39933c511c2d2f21e3be0774232282c1c2431844 and then we are good to go.

**Comment by berezovskyi** [comment]
@neormx

**Comment by neormx** [comment]
It looks good to me @berezovskyi.

**Pull request merged**